common/pom.xml

```diff
@@ -22,5 +22,56 @@
             <groupId>org.jsoup</groupId>
             <artifactId>jsoup</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-webmvc</artifactId>
+            <version>4.3.14.RELEASE</version>
+            <scope>compile</scope>
```

Comment:
@Simsteve7 This scope isn't necessary. Same for dependencies below (except servlet-api)